test/itensor.jl

```diff
@@ -609,6 +609,19 @@ end
     for ii ∈ 1:dim(i), jj ∈ 1:dim(j), kk ∈ 1:dim(k)
       @test A[k=>kk,i=>ii,j=>jj]==permA[i=>ii,j=>jj,k=>kk]
     end
+
+    Aorig = deepcopy(A)
```

Comment:
Is the `deepcopy` here necessary? I think our regular `copy(::ITensor)` is already a "deep copy" in the sense that it copies the indices and storage. Partially I'm asking because this started to make me wonder about the copying behavior of ITensors...

Reply:
Will change, thanks